### ISSUE TITLE
All Travis-CI jobs now failing

### ISSUE THREAD

**Issue by goneall** [spdx/license-list-XML#1176]
Travis CI jobs are failing with the following error:

```
Processing License List.05:12:45.028 [main] ERROR org.apache.jena.riot - loading remote context failed: https://wking.github.io/fsf-api/schema/licenses.jsonld
Error parsing FSF license data
```

**Comment by goneall** [comment]
There has not been any change to the tools or scripts.  The failures started around Jan 31 with PR #1174 followed by the same error in PR #1175 

The error indicates the file at https://wking.github.io/fsf-api/schema/licenses.jsonld can not be loaded when running in a Travis environment.

The file can be accessed by me locally in a browser.

Running the same scripts locally on my machine works fine.

Running the scripts on travis-ci.com instead of travis-ci.org from a separate repository yield the same error.

The errors are not related to the pull requests - it can be duplicated running the scripts against master which has previously succeeded.

**Comment by goneall** [comment]
Resolved by PR #1177 

Also migrated from travis-ci.org to travis-ci.com
